src/main/java/com/example/repick/global/oauth/KakaoUserService.java

```diff
@@ -82,7 +83,9 @@ private KakaoUserDto handleKakaoResponse(String responseBody) throws JsonProcess
 
         String thumbnailImage = jsonNode.get("kakao_account").get("profile").get("thumbnail_image_url").asText();
 
-        return KakaoUserDto.of(id, email, nickname, thumbnailImage);
+        String gender = jsonNode.get("kakao_account").get("gender").asText();
```

Comment:
`jsonNode.get("kakao_account").get("gender")` 가 null 이라면 다음 에러가 발생합니다:

```
Cannot invoke "com.fasterxml.jackson.databind.JsonNode.asText(String)" because the return value of "com.fasterxml.jackson.databind.JsonNode.get(String)" is null
```

소셜로그인 시 선택 값(성별)을 동의하지 않으면 다음 에러가 발생합니다! 구글과 네이버도 선택 동의라면 해결해야 할 것 같습니다

Reply:
감사합니다 해결했습니다😉